clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -212,6 +215,36 @@ public boolean isVersionSupported(short apiVersion) {
         return apiVersion >= oldestVersion() && apiVersion <= latestVersion();
     }
 
+    public boolean isVersionEnabled(short apiVersion, boolean enableUnstableLastVersion) {
+        // ApiVersions API is a particular case that we always accept any, even
```

Comment:
It would be helpful to clarify the reason for this.

Reply:
Let me extend the comment.